systemtest/src/test/java/io/strimzi/systemtest/operators/FeatureGatesST.java

```diff
@@ -338,21 +326,15 @@ private void setupClusterOperatorWithFeatureGate(String extraFeatureGates) {
      *                         if the rolling update has been completed across all replicas     .
      */
     private void rollKafkaNodePoolWithActiveProducerConsumer(TestStorage testStorage, int kafkaReplicas) {
-        final String producerName = testStorage.getProducerName() + "-rolling";
-        final String consumerName = testStorage.getConsumerName() + "-rolling";
-
-        // setup clients
-        KafkaClients clients = new KafkaClientsBuilder()
-            .withProducerName(producerName)
-            .withConsumerName(consumerName)
-            .withBootstrapAddress(KafkaResources.plainBootstrapAddress(testStorage.getClusterName()))
-            .withTopicName(testStorage.getTopicName())
-            .withMessageCount(testStorage.getMessageCount())
-            .withDelayMs(500)
-            .withNamespaceName(testStorage.getNamespaceName())
+
+        resourceManager.createResourceWithWait(KafkaTopicTemplates.topic(testStorage.getClusterName(), testStorage.getContinuousTopicName(), 3, 3, 2, testStorage.getNamespaceName()).build());
+
+        // setup clients to communicate for ~2 minutes
+        KafkaClients clients = ClientUtils.getContinuousPlainClientBuilder(testStorage)
+            .withMessageCount(120)
```

Comment:
again, magic number
Even for the continuous message count I would create some constant (and I mean this one and the default one - 300 messages)

Reply:
the new variable and respective getter  testStorage.getContinuousMessageCount are introduced to reflect situation with messageCount.